doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+                        <div class="panel-heading">
+                            <h4 class="panel-title">
+                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseFour">
+                                <span class="glyphicon"></span>
+                                Answers
+                                </a>
+                            </h4>
+                        </div>
+                        <div id="collapseFour" class="panel-collapse collapse">
+                            <div class="panel-body">
+                                <div class="row">
+                                    <ol>
+                                        <font color="green">
+                                        '?' means it is not under version control so we must add it.
+                                        <br/>
+                                        <font color="black">fcm add my_goodbye_script.py</font>
+                                        <br/>
+                                        ' M' means the properties are modified and no action is needed by us.
+                                        </font>
+                                    </ol>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <pre>
+fcm ci
+<font color="blue">
+[info] vi: starting commit message editor...
+Change summary:
+--------------------------------------------------------------------------------
+[Root   : file:///$HOME/fcm_tutorial/repository]
+[Project: specifics]
+[Branch : branches/dev/$LOGNAME/branch_propset_exe]
+[Sub-dir: ]
+
+A       src/my_goodbye_script.py
+ M      src/my_hello_script.py
+--------------------------------------------------------------------------------
+Commit message is as follows:
+--------------------------------------------------------------------------------
+Changed the properties of src/my_hello_script.py and added a new script.
+--------------------------------------------------------------------------------
+Would you like to commit this change?
+Enter "y" or "n" (or just press &lt;return&gt; for "n"): <font color="black">y</font>
+Adding         src/my_goodbye_script.py
+Sending        src/my_hello_script.py
+Transmitting file data .
+Committed revision 56.
+Updating '.':
+At revision 56.
+</font>
+</pre>
+                Now we have changed the executable properties on the 
+                <samp>my_hello_script.py</samp> script.
+                <br/>
+                If we checked the branch it would have the correct properties.
+                Note that as the trunk has the wrong properties for 
+                <samp>my_hello_script.py</samp> then all of the other branches created from it
+                will have the wrong properties.
+                <br/>
+                We should merge our change into the trunk to correct this.
```

Comment:
Do you want them to actually do this in their example? If so, I'd add a "Do this now." sentence on the end.